lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -144,6 +155,45 @@ bool PatchInOutImportExport::runOnModule(Module &module) {
   return true;
 }
 
+// =====================================================================================================================
+// Mark the 'done' flag to the very last position export instruction.
+//
+// @param [in/out] func : LLVM function to be run on
+// @param [in] pdt : The PostDominatorTree of the \p func
+void PatchInOutImportExport::markExportDone(Function *func, PostDominatorTree &pdt) {
+  std::vector<CallInst *> expInstrs;
+
+  Function *expDecl = m_module->getFunction("llvm.amdgcn.exp.f32");
```

Comment:
The old code also handled "llvm.amdgcn.exp.compr.v2f16". Is that not needed?

Reply:
I think that is used for fragment shader which has been moved out.